ratebeer/ratebeer.py

```diff
@@ -42,17 +41,30 @@ class RateBeer(object):
 
     .. code:: python
 
-        >>> rb = RateBeer()
-        >>> rb.search("summit extra pale ale")
-        {'beers': [{'name': [u'Summit Extra Pale Ale'],
-            'num_ratings': 678,
-            'rating': 73  ,
-            'url': u'/beer/summit-extra-pale-ale/7344/'}],
-        'breweries': []}
+        >>> import ratebeer
+        >>> rb = ratebeer.RateBeer()
+        >>> results = rb.search("summit extra pale ale")
+            {'beers': [Beer('/beer/summit-extra-pale-ale/7344/'),
+                       Beer('/beer/summit-extra-pale-ale--rose-petals/317841/')],
+             'breweries': []}
+        >>> results['beers'][0].__dict__
+            {'_has_fetched': True,
+             'abv': 5.1,
+             'brewery': 'Summit Brewing Company',
+             'brewery_country': 'USA',
+             'brewery_url': '/brewers/summit-brewing-company/1233/',
+             'calories': 153.0,
+             'description': 'Summit Extra Pale Ale is not a beer brewed only for beer snobs. Just the opposite. It\x92s a beer for everyone to enjoy: construction workers, stock brokers, farmers, sales people, clerks, teachers, lawyers, doctors, even other brewers. Its light bronze color and distinctly hoppy flavor have made it a favorite in St. Paul, Minneapolis and the rest of the Upper Midwest ever since we first brewed it back in 1986.',
+             'name': 'Summit Extra Pale Ale',
+             'num_ratings': 698,
+             'overall_rating': 68,
+             'style': 'American Pale Ale',
+             'style_rating': 59,
+             'url': '/beer/summit-extra-pale-ale/7344/',
+             'weighted_avg': 3.27}
 
     See the full README at https://github.com/alilja/ratebeer
     """
-    VERSION = "2.1"
 
     def search(self, query):
```

Comment:
The RateBeer search page has been updated (as of August 6th), meaning that the search function has broken.  I can try to work on it later, but for the time being it is out of order.